.travis.yml

```diff
@@ -4,7 +4,7 @@ language: node_js
 branches:
   only:
     - master
-    - 3.6
+    - next
```

Comment:
Accidentally left this change in the previous merge from `3.6`, so I reverted it here. But really we should just delete `.travis.yml` since we don't use Travis and therefore don't maintain this file.